### ISSUE TITLE
var lookup doesn't work with `#` in identifier 

### ISSUE THREAD

**Issue by zarkone** [xapix-io/axel-f#62]
```clojure
(let [u "localhost-api-users#cache-lookup"]
  (axel-f/run u {u {:body 12}})) => nil
```

**Comment by mreinsch** [comment]
will be part of upcoming 2.0.0

**Comment by mreinsch** [comment]
https://github.com/xapix-io/axel-f/pull/66

**Comment by zarkone** [comment]
fixed in 2.0.0!
